pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTest.java

```diff
@@ -211,6 +208,13 @@ public void testSqlWithExceptionMsgChecker(String sql, String exceptionMsg) {
           _reducerScheduler, null, false);
     } catch (RuntimeException rte) {
       Assert.assertTrue(rte.getMessage().contains("Received error query execution result block"));
+      // TODO: The actual message is (usually) something like:
```

Comment:
I don't know if this is actually what we want to test or not. 

I think there is a race condition and sometimes the message we get is directly the one from the original error block (Illegal Json Path...) and other times we receive a new error block created during the op chain cancellation, which did not contain the message. In order to pass the test I added the original message to that cancellation block, but it may just a hack that makes the test pass but does not implement the logic we actually wanted to test